packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/cdk-cdk-diff--import-existing-resources-shows-import.integtest.ts

```diff
@@ -0,0 +1,45 @@
+import { integTest, withSpecificFixture } from '../../lib';
+
+jest.setTimeout(2 * 60 * 60_000); // Includes the time to acquire locks, worst-case single-threaded runtime
+
+integTest(
+  'cdk diff --import-existing-resources show resource being imported',
+  withSpecificFixture('import-app', async (fixture) => {
+    // GIVEN
+    await fixture.cdkDeploy('base-1', {
+      modEnv: {
+        VERSION: 'v1',
+      },
+    });
+
+    // THEN
+    let diff = await fixture.cdk(['diff --import-existing-resources', fixture.fullStackName('base-1')], {
+      modEnv: {
+        VERSION: 'v2',
+      },
+    });
+
+    // Assert there are changes
+    expect(diff).not.toContain('There were no differences');
+    expect(diff).toContain('[←]');
+    expect(diff).toContain('import');
+
+    // THEN
+    diff = await fixture.cdk(['diff', fixture.fullStackName('base-1')], {
+      modEnv: {
+        VERSION: 'v2',
+      },
+    });
+
+    // Assert there are changes
```

Comment:
Like this?
```suggestion
    // Assert there are no changes
```

For my understanding: Is this test just trying to show the difference in diff output between `diff` and `diff --import-existing-resources`?

Reply:
Yes, the first diff expects to show resource being imported with `[←]` and `import` substring. The second diff expects `[+]` to make sure `--import-existing-resoruces` is working.